tests/test_jira.py

```diff
@@ -316,3 +316,20 @@ def test_fail():
     """)
     result = testdir.runpytest('--jira')
     assert_outcomes(result, 0, 0, 0, xfailed=1)
+
+
+def test_docstr_marker_xfail(testdir):
+    '''docstring with jira issue id'''
+    testdir.makeconftest(CONFTEST)
+    testdir.makepyfile("""
+        import pytest
+
+        @pytest.mark.jira("ORG-1412", run=True)
```

Comment:
Topic says that "Add test with jira issue id in docstr", but I can see using both: decorator, and doc-string as well.